tests/conftest.py

```diff
@@ -6,6 +6,7 @@
 import pytest
 
 from . import http_test_server
+from .fixtures_jlap import package_server, package_repository_base  # NOQA
```

Comment:
The JLAP specific fixtures should be moved into `conda.testing.gateways.fixtures` and the server code in there into `conda.testing.gateways.jlap` since they are autoloaded with the pytest_plugins attribute below, no need to import them.

Reply:
I had trouble getting that to happen for session-scoped fixtures but we can try again.

Reply:
Thank you!